reference/enterprise-api-ref/health-diagnostic.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Health diagnostic API
+published: true
+tags: [reference, enterprise, API, reporting, URI, health]
+---
+
+This API provides access to health diagnostic information.
+
+## Get health diagnostic status
+
+**URI:** https://hub.cfengine.com/api/health-diagnostic/status
+
+**Method:** GET
+
+**Example response:**
+
+```
+{
+    "hostsNeverCollected": 1,
+    "hostNotRecentlyCollected": 0,
+    "hostsUsingSameIdentity": 0,
+    "agentNotRunRecently": 2,
+    "lastAgentRunUnsuccessful": 0,
+    "totalFailed": 3,
+    "total": "50642"
```

Comment:
What does total include/not include (I'm thinking about hosts not reported)

Reply:
**total** is total host count, the same name was in Mission portal, so I didn't change it. Good note to rename this. I'll do it.